uobtheatre/productions/migrations/0013_alter_production_facebook_event.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.9 on 2021-12-11 13:49
```

Comment:
I'm going to add this migration into the refund branch so can we shove this in there. We can probably also do some squashing

Reply:
I also can't find the migration for post_Image but it seems thats changed as welll. I think the best plan is to: 
- delete all migrations in this branch
- merge with dev 
- regenerate migrations